src/Service/ActivityPub/Wrapper/DeleteWrapper.php

```diff
@@ -28,15 +33,25 @@ public function __construct(
         'to' => 'mixed',
         'cc' => 'mixed',
     ])]
-    public function build(ActivityPubActivityInterface $item, string $id): array
+    public function build(ActivityPubActivityInterface $item, string $id, User $deletingUser = null): array
     {
         $item = $this->factory->create($item);
 
+        $userUrl = $item['attributedTo'];
+
+        if (null !== $deletingUser) {
+            if (null !== $deletingUser->apId) {
```

Comment:
Maybe add some comments for future reference, including for yourself ;)

Reply:
true

Reply:
But you didn't add comments